src/main/resources/reference.conf

```diff
@@ -37,4 +37,36 @@ hmrc-timeout-dialog {
   defaultCountdownInSeconds = 120
 }
 
+pta-account-menu {
+  pertax-frontend {
+    host = "http://localhost:9232"
+    host = ${?platform.frontend.host}
+  }
+
+  tracking-frontend {
+    host = "http://localhost:9100"
+    host = ${?platform.frontend.host}
+  }
+
+  account-home = {
+    host = ${pta-account-menu.pertax-frontend.host}
+    path = ${pta-account-menu.account-home.host}"/personal-account"
+  }
+
+  messages = {
+    host = ${pta-account-menu.pertax-frontend.host}
+    path = ${pta-account-menu.messages.host}"/personal-account/messages"
+  }
+
+  check-progress = {
+    host = ${pta-account-menu.tracking-frontend.host}
+    path = ${pta-account-menu.check-progress.host}"/track"
+  }
+
+  your-profile = {
+    host = ${pta-account-menu.pertax-frontend.host}
+    path = ${pta-account-menu.your-profile.host}"/personal-account/your-profile"
```

Comment:
@kyle-bowden tiny thing, but could we rename these from path to href, matches the config vars then now that it has the host in it too

Reply:
Sure, i will make that change.